iocore/hostdb/HostDB.cc

```diff
@@ -989,65 +1051,50 @@ HostDBContinuation::lookup_done(IpAddr const &ip, const char *aname, bool around
     case TTL_OBEY:
       break;
     case TTL_IGNORE:
-      ttl_seconds = hostdb_ip_timeout_interval;
+      ttl_seconds = ts_seconds(hostdb_ip_timeout_interval);
       break;
     case TTL_MIN:
-      if (hostdb_ip_timeout_interval < ttl_seconds) {
-        ttl_seconds = hostdb_ip_timeout_interval;
+      if (ts_seconds(hostdb_ip_timeout_interval) < ttl_seconds) {
+        ttl_seconds = ts_seconds(hostdb_ip_timeout_interval);
       }
       break;
     case TTL_MAX:
-      if (hostdb_ip_timeout_interval > ttl_seconds) {
-        ttl_seconds = hostdb_ip_timeout_interval;
+      if (ts_seconds(hostdb_ip_timeout_interval) > ttl_seconds) {
```

Comment:
can the macro **_ts_seconds_**  be used when assigning **hostdb_ip_timeout_interval**, rather than on every read?